Neos.Flow/Classes/Mvc/ActionResponse.php

```diff
@@ -153,32 +151,79 @@ public function deleteCookie(string $cookieName): void
      * @param string $parameterName
      * @param mixed $value
      * @return void
-     * @api
+     * @deprecated since Flow 7.0 use setHttpHeader or replaceHttpResponse instead. See also #linkToDocs
```

Comment:
Yeah, I would like to add a link to the rendered release notes with the explanation here, so I put the placeholder in place. Maybe we can remove it for now and just not forget it (and if we do, we're at least not stuck with a strange hashtag).
```suggestion
     * @deprecated since Flow 7.0 use setHttpHeader or replaceHttpResponse instead. For now this will still work with $componentClassName of "SetHeaderComponent" or "ReplaceHttpResponseComponent" only.
```